src/cosl/grafana_dashboard.py

```diff
@@ -21,19 +23,68 @@ class GrafanaDashboard(str):
 
     @staticmethod
     def _serialize(raw_json: Union[str, bytes]) -> "GrafanaDashboard":
-        if not isinstance(raw_json, bytes):
-            raw_json = raw_json.encode("utf-8")
-        encoded = base64.b64encode(lzma.compress(raw_json)).decode("utf-8")
-        return GrafanaDashboard(encoded)
+        warnings.warn(
+            "GrafanaDashboard._serialize is deprecated; use LZMABase64.compress(json.dumps(...)) instead.",
```

Comment:
How long do we support deprecated code in cos-lib for?

Reply:
At this point we'll just need to remember to upkeep such things on a major version bump.